### DISCUSSION TITLE
Dinamic provisioning

### DISCUSSION THREAD

**Discussion by xipi3000** [arkmq-org/arkmq-org-broker-operator#754] [Q&A]
Hi, is there any way to use daemonsets instead of statefull sets? I have a dinamically growing and shrinking kubernetes cluster and i want the artemis cloud to downscale or upscale automatically too, without the size parameter. Also how does the cluster communicate to find nodes without multicasting?

**Comment by brusdev** [comment]
Hi @xipi3000 there is no way to use daemonsets instead of statefull sets. ArtemisCloud operator has only a limited support for HPA (autoscale), for further details see #124. Could you describe why your use case require to downscale or upscale automatically?

The broker pods use JGroups DNS_PING to find nodes without multicasting, see https://github.com/artemiscloud/activemq-artemis-broker-kubernetes-image/blob/main/modules/activemq-artemis-launch/added/jgroups-ping.xml
